lib/hexpm/accounts/single_use_token.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Hexpm.Accounts.SingleUseToken do
+  use Hexpm.Schema
+
+  @token_size 32
+  @allowed_types ["github_merge_token"]
```

Comment:
I'm sure you discussed this with Eric, but are you planning to need more types in the (near) future?

Reply:
TBH, I haven't discussed it yet. I would like the opinion from both of you on what approach we should use for those tokens. @supersimple @ericmj

Reply:
Will we reuse this schema for other south providers?

Reply:
I believe it could be used without problems